app/src/organisms/ODD/QuickTransferFlow/utils/generateQuickTransferArgs.ts

```diff
@@ -425,6 +425,22 @@ export function generateQuickTransferArgs(
         commandCreatorFnName: 'transfer',
         sourceWells,
         destWells,
+        aspirateDelay:
+          quickTransferState.delayAspirate != null
+            ? {
+                seconds: quickTransferState.delayAspirate?.delayDuration,
```

Comment:
nit
probably we can remove `?` since you did null check already
```suggestion
                seconds: quickTransferState.delayAspirate.delayDuration,
```